src/components/arbitrum/feeds/SequencerFeedDetail.tsx

```
@@ -0,0 +1,120 @@
+          }
+          //   if (isStepSkipped(index)) {
+          //     stepProps.completed = false;
+          //   }
+          return (
+            <Step key={label}>
+              <StepLabel>{label}</StepLabel>
+            </Step>
+          );
+        })}
+      </Stepper>
+      {childArray && childArray[activeStep]}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)